src/types/common.ts

```diff
@@ -0,0 +1,174 @@
+//** TODO: Resolve duplication with Flow */
+export const GLYPH_PROPS_VALID_TYPES: Record<string, Record<string, string>> = {
```

Comment:
Can you not do this:

```
export const GLYPH_PROPS_VALID_TYPES: Record<string, Record<string, string>> = Flow.getGlyphProps.validTypes
```

Reply:
If this works, then I'd say we can put it in `Note`.

Either way, this is just a comment, we can do this in a followup PR.

Reply:
I would wait to see other things that come out of table.